libraries/helpers.rb

```diff
@@ -57,7 +57,7 @@ def activation_keys
     end
 
     def registered_with_rhsm?
-      cmd = Mixlib::ShellOut.new('subscription-manager status')
+      cmd = Mixlib::ShellOut.new('subscription-manager status', env: { LANG: 'en_US' })
```

Comment:
If possible it would be nice to have `en_US` as an attribute or something so if at some point we needed to override it we could.